talentsearch/resources/js/components/SearchDashboard.tsx

```diff
@@ -38,7 +48,7 @@ export const SearchDashboard: React.FC = () => {
       href="/"
       text={intl.formatMessage(messages.menuHome)}
     />,
```

Comment:
You probably want to override the isActive property for the landing page, so the link doesn't stay always bolded.